lib/src/icon/Icon.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import styled from "styled-components";
+import IconPropsType from "./types";
+
+const DxcIcon = ({ icon }: IconPropsType): JSX.Element => {
+  let filled = false;
+  let iconName = icon;
+  if (icon.startsWith("filled_")) {
+    filled = true;
+    iconName = icon.replace(/filled_/g, "");
+  }
+  return <IconContainer role="img" aria-label={icon} filled={filled} icon={iconName} />;
+};
+
+const IconContainer = styled.span<{
+  icon: string;
+  filled: boolean;
+}>`
+  font-family: "Material Symbols Outlined";
+  font-weight: normal;
+  font-style: normal;
+  font-size: 24px;
+  line-height: 1;
+  letter-spacing: normal;
+  text-transform: none;
+  display: inline-block;
+  white-space: nowrap;
+  word-wrap: normal;
+  direction: ltr;
+  -webkit-font-feature-settings: "liga";
+  -webkit-font-smoothing: antialiased;
+  font-variation-settings: ${(props) => (props.filled ? "'FILL' 1" : "'FILL' 0")};
+  ::before {
+    content: "${(props) => props.icon}";
+  }
```

Comment:
The following rules could be added for extra browser compatibility
```
/* Support for Safari and Chrome. */
text-rendering: optimizeLegibility;

/* Support for Firefox. */
-moz-osx-font-smoothing: grayscale;

/* Support for IE. */
font-feature-settings: 'liga';
```
[Google fonts](https://developers.google.com/fonts/docs/material_icons?hl=es-419)